pages/api/inventory/UpdateItem.js

```diff
@@ -60,7 +60,7 @@ export default async function(req,res) {
     }
 
     FIELDS.forEach(field => {
-      if (body[field]) {
+      if (body[field] || body[field] == 0) {
```

Comment:
i think  `if ... === undefined` is a lil bit better

just on the weird off chance that we want to name an item "false" or smth LOL

Reply:
thanks for fixing it so fast though 🤩

Reply:
Ooo you right just changed it to that